.github/workflows/scripts/push.js

```diff
@@ -0,0 +1,35 @@
+module.exports = async ({ core, exec, context }, token) => {
+  const { Octokit } = require("@octokit/core")
+  const octokit = new Octokit({ auth: token })
+  const checkoutPageDir = "gh-pages"
+  const helm = {
+    owner: process.env.OWNER,
+    repo: process.env.REPO,
+    ref: process.env.REF,
+    charts: JSON.parse(process.env.CHARTS)
+  }
+
+  const headerToken_b64 = Buffer.from(`x-access-token:${token}`).toString('base64')
+  core.setSecret(headerToken_b64)
+
+  try {
+    await exec.exec('git', ['remote', '-v'], { cwd: checkoutPageDir })
+    await exec.exec('git', ['status'], { cwd: checkoutPageDir })
+    await exec.exec('git', ['config', '--local', '--unset-all', 'http.https://github.com/.extraheader'], { cwd: checkoutPageDir })
+    await exec.exec('git', ['config', '--local', 'http.https://github.com/.extraheader', `AUTHORIZATION: basic ${headerToken_b64}`], { cwd: checkoutPageDir })
+    await exec.exec('git', ['config', '--local', 'user.name', 'G-Research charts'], { cwd: checkoutPageDir })
+    await exec.exec('git', ['config', '--local', 'user.email', 'charts@gr-oss.io'], { cwd: checkoutPageDir })
+
+    await exec.exec('helm', ['repo', 'index', '.'], { cwd: checkoutPageDir })
+    await exec.exec('git', ['add', 'index.yaml'], { cwd: checkoutPageDir })
+
+    await exec.exec('git', ['status'], { cwd: checkoutPageDir })
+    await exec.exec('git', ['commit', '-m', `Publish helm chart to ${context.payload.repository.owner.login}/${context.payload.repository.name}`, '--verbose'], { cwd: checkoutPageDir }) 
```

Comment:
Suggestion: put more information in the commit message, perhaps about when/how the script was invoked?

Reply:
as this is the only way to publish this chart (after merging) I don't see a reason why in the commit message we would have `when/how` answer, but reasonably looking IMHO it was missing a `ref` a.k.a `version` of that chart which was published so in future in commit messages you can search explicit version
FYI I'm pushing that in the following commit